grakn-dist/src/graql

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+#
+# Grakn - A Distributed Semantic Database
+# Copyright (C) 2016  Grakn Labs Limited
+#
+# Grakn is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Grakn is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# <http://www.gnu.org/licenses/gpl.txt>.
+
+# ================================================
+# global variables
+# ================================================
+
+# Grakn globals
+# TODO: clean up
+if [ -z "${GRAKN_HOME}" ]; then
+    [[ $(readlink $0) ]] && path=$(readlink $0) || path=$0
+    GRAKN_HOME=$(cd "$(dirname "${path}")" && pwd -P)
+    GRAKN_CONFIG="${GRAKN_HOME}/conf/grakn.properties"
+    GRAKN_PS=/tmp/grakn-engine.pid
+    GRAKN_STARTUP_TIMEOUT_S=120
+fi
+
+# ================================================
+# common helper functions
+# ================================================
+update_classpath() {
+  # Define CLASSPATH, exclude slf4j as we use logback
+  for jar in "${GRAKN_HOME}"/services/lib/*.jar; do
+      if [[ $jar != *slf4j-log4j12* ]] ; then
+          CLASSPATH="$CLASSPATH":"$jar"
+      fi
+  done
+
+  # Add path containing grakn.properties and logback.xml
+  CLASSPATH="$CLASSPATH":"${GRAKN_HOME}"/conf
+  CLASSPATH="$CLASSPATH":"${GRAKN_HOME}"/services/grakn
+}
+
+graql_console() {
+  echo "Starting Graql console..."
+  java -cp ${CLASSPATH} -Dgrakn.dir="${GRAKN_HOME}/services" ai.grakn.graql.GraqlShell ${1+"${@:2}"}
+}
+
+graql_help() {
+  echo "
+Usage:
+graql COMMAND
+
+COMMAND:
+console  Starts a REPL console for running Graql queries. Defaults to connecting to http://localhost
+migrate  Runs migration from a file
+help     Prints this message"
+}
+
+# =============================================
+# main routine
+# =============================================
+
+update_classpath
+
+case "$1" in
+  console)
+    graql_console
```

Comment:
yes we have that for graql and for migration, they can be invoked via `graql console help` and `graql migrate help`, respectively.

Different than `grakn`, the help function for graql and migration are already built-in in the Java program since way back (not very consistent, but I guess it'll do for now)